### ISSUE TITLE
Installation problem (windows 10)

### ISSUE THREAD

**Issue by Elgeneinar** [PokeAPI/pokeapi#558]
I used choco install make, and then ran make install. 
I get this error:

ERROR: Command errored out with exit status 1:
     command: 'c:\python39\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\[user]\\AppData\\Local\\Temp\\pip-install-sok1y4vt\\psycopg2_3feed76d307847c3abed3ce0a85a8acd\\setup.py'"'"'; __file__='"'"'C:\\Users\\[user]\\AppData\\Local\\Temp\\pip-install-sok1y4vt\\psycopg2_3feed76d307847c3abed3ce0a85a8acd\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg'
         cwd: C:\Users\[user]\AppData\Local\Temp\pip-install-sok1y4vt\psycopg2_3feed76d307847c3abed3ce0a85a8acd\
    Complete output (23 lines):
    running egg_info
    creating C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg\psycopg2.egg-info
    writing C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg\psycopg2.egg-info\PKG-INFO
    writing dependency_links to C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg\psycopg2.egg-info\dependency_links.txt
    writing top-level names to C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg\psycopg2.egg-info\top_level.txt
    writing manifest file 'C:\Users\[user]\AppData\Local\Temp\pip-pip-egg-info-eohw0ieg\psycopg2.egg-info\SOURCES.txt'

    Error: pg_config executable not found.

    pg_config is required to build psycopg2 from source.  Please add the directory
    containing pg_config to the $PATH or specify the full executable path with the
    option:

        python setup.py build_ext --pg-config /path/to/pg_config build ...

    or with the pg_config option in 'setup.cfg'.

    If you prefer to avoid building psycopg2 from source, please install the PyPI
    'psycopg2-binary' package instead.

    For further information please check the 'doc/src/install.rst' file (also at
    <http://initd.org/psycopg/docs/install.html>).

    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Comment by Naramsim** [comment]
Hi @Elgeneinar, I personally don't use directly Python for building `pokeapi`, I use the docker setup.

If you look online for the query _Error: pg_config executable not found._ you can find many results on SO. In theory, you need to install a library on your machine.

Maybe you can follow some of those guides and then let us know how you overcame the issue.

But I strongly suggest using the Docker setup, if you have Docker.

**Comment by Elgeneinar** [comment]
> 
> 
> Hi @Elgeneinar, I personally don't use directly Python for building `pokeapi`, I use the docker setup.
> 
> If you look online for the query _Error: pg_config executable not found._ you can find many results on SO. In theory, you need to install a library on your machine.
> 
> Maybe you can follow some of those guides and then let us know how you overcame the issue.
> 
> But I strongly suggest using the Docker setup, if you have Docker.

Oh, I do have docker, I just followed the install instructions as written in the Readme. I very possibly missed something about the docker, so I will look further tomorrow. Thanks for your reply.

**Comment by C-Garza** [comment]
I remember having this issue when I first set it up. I think it was having some problem with their pip packages or something, so I just followed the docs on their site to install it. 

Leaving a link to their docs in case anyone else comes across this issue. [Psycopg Docs](https://www.psycopg.org/docs/install.html#change-in-binary-packages-between-psycopg-2-7-and-2-8).

**Comment by Elgeneinar** [comment]
I got it to build today. I first explored those Psycopg Docs linked by @C-Garza, however even when the psycopg2_binary got successfully installed, it did not affect the error. 
I then tried the Docker setup, which initially failed due to a port access error, so I opened the `docker-compose.yml` file and set the port to `81:80` instead of `80:80`.
After doing this and running `make docker-setup` again, it built without issues.
Using the JSON view chrome extension was helpful when looking at the data in the browser.
Thanks for helping me out with this 👍
